extensions/amazon-lambda-xray/runtime/pom.xml

```diff
@@ -46,15 +46,20 @@
             </plugin>
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <annotationProcessorPaths>
-                        <path>
-                            <groupId>io.quarkus</groupId>
-                            <artifactId>quarkus-extension-processor</artifactId>
-                            <version>${project.version}</version>
-                        </path>
-                    </annotationProcessorPaths>
-                </configuration>
+                <executions>
+                    <execution>
+                        <id>default-compile</id>
```

Comment:
majority of the changes is the change of config here where only diff seem to be giving it an id. is that essential or can we greatly simplify the diff?

Reply:
The point is not to give it an id: I avoid running the annotation processor for the tests. This will attach the annotation processor to the `default-compile` goal run.

Reply:
ah...so to save time and redundancy. gotcha.  

could be a separate pr to make this one much simpler but if its the only thing that happens in ~500 files I guess we'll be good :)